src/landOnFunctions.js

```diff
@@ -1,30 +1,119 @@
 export function standardProperty(activePlayer, emitter){
     if(this.ownerID === null){
-        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
+        emitter.once(`confirmBuy`, (io, bool) => {
+            if(bool){
+                if(activePlayer.money < this.cost){
+                    io.to(activePlayer.socketID).emit('msg', `insufficient funds, purchase of ${this.nameStr} cancelled`);//IDEA: keep track of round info to send and make game logs
+                }
```

Comment:
Typically, we put else on the same line as the closing brace it's elsing so it's clear which block it's the else of.

```
if (thing) {
    /// Stuff
} else {
    // Other Stuff
}